packages/web/src/components/stem-files-modal/StemFilesModal.tsx

```diff
@@ -6,11 +6,9 @@ import {
   StemUpload,
```

Comment:
we don't do anything lossless related in this file anymore, because it's only use when the feature flag is false

Reply:
hm were we doing lossless stuff before? or did we not have to do anything in this file to begin with?

Reply:
so i started messing with this file to do lossless stuff, then realized i should use the same menu fields in the edit flow as those in the upload flow, so these changes were checked in for a bit, but i went a different/better route.
good q tho

Reply:
StemFilesView is the one we care about for lossless stuff, which is inside the menu fields :)